docs/integrations/app-development/github.md

```diff
@@ -254,3 +254,54 @@ Use this dashboard to:
 * Identify any harmful file types added by users.
 
 <img src={useBaseUrl('img/integrations/app-development/Github-User-Activity.png')} alt="GitHub-Overview" />
+
+### GHAS - Advanced Security Overview
+
+The **GHAS - Advanced Security Overview** dashboard provides an overview of GHAS metrics across Dependabot, secret scanning, and code scanning alerts.
+
+Use this dashboard to:
+
+* Monitor open alerts 
+* Monitor alerts by severity 
+* Review recently closed alerts
+ 
+<img alt="undefined" class="default" src="https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/GitHub/GHAS-Advanced-Security-Overview.png" />
+
+### GHAS - Secret Scanning Alerts 
+
+**Use this dashboard to:**
+
+* Monitor MTTR
+* Quantify secrets found and fixed
+* Check secrets by type and repository
+* Compare secrets and found to secrets in fixed ratios
+
+<img alt="undefined" class="default" src="https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/GitHub/GHAS-Secret-Scanning-Alerts.png"/>
+
+###GHAS - Code Scanning Alerts 
```

Comment:
Hi @stacykor, thank you for popping in the S3 links here! One thing -- in line 281 here, you need a space after the `###`. Without this, formatting is lost and header doesn't populate in menu.